components/supervisor-api/info.proto

```diff
@@ -78,4 +78,26 @@ message WorkspaceInfoResponse {
 
     // ide_port is the port on which the IDE is to be run
     uint32 ide_port = 14;
+
+    message WorkspaceClass {
```

Comment:
let's not copy all api from server, it is rather internal thing, and here we are quite public, only add what we really need for now: `id`, `name`, `description`